server/helpers/dates.js

```diff
@@ -8,3 +8,5 @@ export const getTodaysDateShortCode = () => format(new Date(), 'ddMMyyyy');
 export const getTodaysDateStandard = () => format(new Date(), 'yyyy-MM-dd');
 
 export const getDateWithTimestamp = () => formatISO(new Date());
+
+export const convertDateToWords = (date) => format(date, 'd MMM yyyy');
```

Comment:
We should use the Long format in emails so that it is more explicit. The short one can be used for other API messages that require a human-readable format.
```suggestion
export const convertDateToLongHumanFormat  = (date) => format(date, 'dddd, Do MMMM YYYY');
export const convertDateToShortHumanFormat = (date) => format(date, 'ddd, MMM D, YYYY');
```